site/content/docs/alerts/10009.md

```diff
@@ -7,11 +7,10 @@ alertcount: 1
 status: beta
 type: alert
 risk: Low
-solution: "Configure the server to prevent such information leaks. For example:
-Under Tomcat this is done via the 'server' directive and implementation of custom error pages.
-Under Apache this is done via the 'ServerSignature' and 'ServerTokens' directives."
+solution: "Configure the server to prevent such information leaks. For example: Under Tomcat this is done via the \"server\" directive and implementation of custom error pages. Under Apache this is done via the \"ServerSignature\" and \"ServerTokens\" directives."
```

Comment:
Is this the intended output?

Reply:
Whatever works is the intended output :wink:
I had a lot of trouble getting the encoding to work in all cases...

Reply:
The tags being removed is a bit worrying though, need to double check that..